Assets/MixedRealityToolkit/_Core/Sharing/AnchorControl/AlignmentState.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Pixie.Core;
+using System;
+using UnityEngine;
+
+namespace Pixie.AnchorControl
+{
+    [Serializable]
+    public struct AlignmentState : IItemState, IItemStateComparer<AlignmentState>
+    {
+        const float MaxPositionRange = 20;
+
+        public AlignmentState(short userID)
+        {
+            UserNum = userID;
+            StateUtils.ShortPosIn(Vector3.zero, out PositionX, out PositionY, out PositionZ, MaxPositionRange);
+            StateUtils.ShortRotIn(Vector3.zero, out RotationX, out RotationY, out RotationZ);
+        }
+
+        short IItemState.Key { get { return UserNum; } }
+
+        public short UserNum;
+
+        public Vector3 Position
```

Comment:
Comments on what these positions and angles are relative to would likely prove helpful.